tap_zuora/discover.py

```diff
@@ -23,18 +24,58 @@
     "UpdatedOn",
 ]
 
-REQUIRED_KEYS = ["Id"] + REPLICATION_KEYS
+UNSUPPORTED_FIELDS_FOR_REST = {
```

Comment:
Are we really sure that these fields are not supported in REST API? Is it mentioned on the official docs? Let's not rely only on test data. 
The reason being if we merge with a minor version bump then these fields won't be replicated in the target destination (if they are present for any of the customer's data).

Reply:
ok, the main reason behind adding these unsupported fields for REST API is we have noticed extraction failures on prod for clients who have selected these fields.Making these fields unsupported during discovery phase, so users won't be able to select them. 
The behaviour would be same for all the accounts, since we are using same WSDL version which happens to be 91.0
and I strongly believe this is not going to break anything at destination side because the extraction itself fails upon selecting these fields.

@dmosorast do you think it is ok to do a minor release with this change?